packages/http/src/mocker/negotiator/__tests__/NegotiatorHelpers.spec.ts

```diff
@@ -302,15 +302,15 @@ describe('NegotiatorHelpers', () => {
     });
 
     it('given response defined should try to negotiate by that response', () => {
-      const code = faker.random.word();
+      const code = faker.random.number();
       const fakeResponse = {
-        code,
+        code: code.toString(),
         contents: [],
         headers: [],
       };
       const desiredOptions = { dynamic: false };
       const fakeOperationConfig: IHttpNegotiationResult = {
-        code,
+        code: code.toString(),
```

Comment:
If we're doing this change, I guess it would be a good idea to also make sure the returned result is a number.

Reply:
I'm not sure what you mean. All the functions called in this test and the function `negotiateOptionsBySepcificCode` return an object where code is a string not a number.

Can you elaborate on what you mean?

Reply:
or do you just mean that the code is parseInt-able?

Reply:
I have checked this out and unfortunately we'll have to leave this as is at the moment, because of the @stoplight/types package.

Reply:
womp womp 😞